app/portainer/components/focusIf.js

```diff
@@ -0,0 +1,20 @@
+angular.module('portainer.app').directive('focusIf', function ($timeout) {
+  return {
```

Comment:
this directive is taken from somewhere, right? we should have a link in comments.